### PR TITLE
DAOS-10217-test: Invalid pool rebuild info rs_state=1 detected after rebuild

### PR DESCRIPTION
**Pull request by dinghwah**
Description:  rebuild will keep retry due to out of space (-1007) error. Update test script per JIRA comment, rs_state could be 0 or 1.

Quick-Functional: true
Test-tag: no_cap
Test-repeat: 10
Signed-off-by: Ding Ho ding-hwa.ho@intel.com

### CODE REVIEW

**Comment by dinghwah** [comment]
Script updates passed. 
1 out of 10 failure was due to DAOS-9028: failed on destroy container DER_IO(-2001): I/O error when pool is full.

**Review by shimizukko — APPROVED** [review]

File: src/tests/ftest/rebuild/no_cap.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def test_rebuild_no_capacity(self):
         pool_checks["pi_ndisabled"] = ">0"
         rebuild_checks["rs_obj_nr"] = ">=0"
         rebuild_checks["rs_rec_nr"] = ">=0"
-        rebuild_checks["rs_state"] = 0
+        rebuild_checks["rs_state"] = ">=0"
```
Comment (daltonbohning) [inline src/tests/ftest/rebuild/no_cap.py:159]: Is it possible for `rs_state` to be something other than 0 or 1? I.e., @wangdi1 mentions 0 or 1 is valid: https://daosio.atlassian.net/browse/DAOS-10217
But is is possible for query to return, for example, `rs_state == 2`?

**Review by daltonbohning — APPROVED** [review]

File: src/tests/ftest/rebuild/no_cap.py (hunk 1 above)
Comment (dinghwah) [reply src/tests/ftest/rebuild/no_cap.py:159]: I think rs_state will not return 2.

Comment (daltonbohning) [reply src/tests/ftest/rebuild/no_cap.py:159]: Thanks. LGTM

**Pull request merged**